demisto_sdk/main.py

```diff
@@ -169,15 +170,32 @@ def unify(**kwargs):
 @click.option(
     '-g', '--use-git', is_flag=True, show_default=True,
     default=False, help='Validate changes using git - this will check your branch changes and will run only on them.')
+@click.option(
+    '-p', '--path', help='Path of file to validate specifically.'
+)
+# @click.option(
+#    '-t', '--file-type', type=click.Choice(
+#        ['integration', 'script', 'playbook', 'incidnet-field', 'indicator-field', 'image', 'release-notes', 'layout',
+#         'description']), help='If you want to validate a specific file, you can specify it\'s type.')
```

Comment:
lets remove this

Reply:
It was there in case we wanted to pre-define the type, but I just added the logic to the regular flow so it is not needed